internal/appsec/config/rules_manager.go

```diff
@@ -114,6 +114,7 @@ func (r *RulesManager) Clone() (clone RulesManager) {
 	for k, v := range r.Edits {
 		clone.Edits[k] = v
 	}
+	clone.BasePath = r.BasePath
```

Comment:
Smells like maybe the `Clone` method should be generated by a tool (so new fields don't get forgotten)?